Bindings/bindings.json

```diff
@@ -1065,7 +1065,7 @@
         },
         {
             "type": "easyTable",
```

Comment:
why is the type still easyTable? We've already released the breaking change to rename that to mobileTable in the runtime.